src/rgw/rgw_tracer.cc

```diff
@@ -0,0 +1,133 @@
+
+#include "rgw_tracer.h"
+
+using jConfig = jaegertracing::Config;
+using jLogger = jaegertracing::logging::Logger;
+
+namespace RGW_Tracing{
+
+    /*
```

Comment:
style: please avoid these kinds of comments
comments should be used only when information is needed in order to understand non trivial code code